xpublish/rest.py

```diff
@@ -55,6 +66,8 @@ def __call__(self, cache_kws=None, app_kws=None):
             raise RuntimeError('This accessor has already been initialized')
         self._initialized = True
 
+        if routers is not None:
+            self._app_routers = normalize_app_routers(routers)
```

Comment:
@benbovy Is there a way to also inject the default routers to each dataset? It seems like this should be a default behaviour then have a way to turn on or off so by default, the data manager get those default routes. Idk, what do you think? Thanks. This is looking great! I'm looking forward to trying it out.

Reply:
@lsetiawan  If `routers` is `None` (default), the following routers, set in the accessor `__init__`, will be included:

```python
self._app_routers = [
    (common_router, {}),
    (base_router, {'tags': ['info']}),
    (zarr_router, {'tags': ['zarr']}),
]
```

If `routers` is a list, none of those routers are included by default, but it is still possible to include some or all of them, e.g.,

```python
import xarray as xr
from fastapi import APIRouter
from xpublish.routers import base_router, common_router

ds = xr.Dataset(...)

my_custom_router = APIRouter()

ds.rest(routers=[base_router, common_router, my_custom_router]).serve()
```

Does it answer your question? Or did I misunderstand your comment?